### PR TITLE
2.) Shader files, LICENSE and README

### PR DESCRIPTION
**Pull request by rnixx**
Number 2 of spitting up the housekeeping PR

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+----------------
+
+- Make shaders directory a python package and add included shader file paths to __init__.py
+
+
+0.0.1.dev0 (2020-05-23)
+-----------------------
+
+- Initial release
+
```
Comment (AndreMiras) [inline CHANGES:11]: Just curious, why not a `CHANGELOG.md` instead?
https://github.com/olivierlacan/keep-a-changelog/blob/v1.1.0/CHANGELOG.md

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Install dependencies.
 
     $ ./bin/pip install Cython==0.29.9
     $ ./bin/pip install kivy==1.11.1
+    $ ./bin/pip install pyyaml
```
Comment (AndreMiras) [inline README.md:21]: I guess at some point we want to migrate to more conventional dependency install method.
Let's keep it in mind for later

**Review by AndreMiras — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (rnixx) [reply README.md:21]: sure, install convenience and instructions is planned with making dependencies optional

File: CHANGES (hunk 1 above)
Comment (rnixx) [reply CHANGES:11]: habitual. no specific reason

**Pull request merged**